bin/node/runtime/Cargo.toml

```diff
@@ -49,6 +49,7 @@ frame-support = { version = "4.0.0-dev", default-features = false, path = "../..
 frame-system = { version = "4.0.0-dev", default-features = false, path = "../../../frame/system" }
 frame-system-benchmarking = { version = "4.0.0-dev", default-features = false, path = "../../../frame/system/benchmarking", optional = true }
 frame-election-provider-support = { version = "4.0.0-dev", default-features = false, path = "../../../frame/election-provider-support" }
+frame-npos-elections-solution-type = { version = "4.0.0-dev", default-features = false, path = "../../../frame/election-provider-support/solution-type" }
```

Comment:
What do you think about re-exporting the macro from `frame-election-provider-support` to avoid the secondary import?